discovery/src/main/java/com/ibm/watson/discovery/v1/Discovery.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019, 2020.
```

Comment:
This file only has copyright changes so it should have been linted by the fix-copyrights script